lib/utils/puppeteer.js

```diff
@@ -54,6 +49,9 @@ module.exports = async (extraOptions = {}) => {
         puppeteer.use(require('puppeteer-extra-plugin-stealth')());
     }
     let browser;
+    if (proxyUri) {
+        options.args.push(`--proxy-server=${await proxyChain.anonymizeProxy(proxyUri)}`);
+    }
```

Comment:
Why has this code been moved here, I think it would be more efficient to put it up there and only execute it once 👀

Reply:
It can't pass this simple test https://github.com/DIYgod/RSSHub/blob/9329a44c80a37b9aec18a233c9e3bdb3f1c7621d/test/utils/puppeteer.js#L83-L90
The launch options observed from puppeteer process are still the `options` defined above.